apps/cms/lib/partial/paragraph/trip_plan_widget.ex

```diff
@@ -0,0 +1,32 @@
+defmodule CMS.Partial.Paragraph.TripPlanWidget do
+  @moduledoc """
+  Represents a widget on the page leading to the Trip Planner.
+  Is also used for the Fare Calculator widget.
+  """
+  import CMS.Helpers,
+    only: [
+      field_value: 2
+    ]
+
+  @type t :: %__MODULE__{
+          title: String.t() | nil,
+          text: String.t() | nil,
+          button_text: String.t() | nil,
+          right_rail: boolean
+        }
+
+  defstruct title: "Plan a trip",
```

Comment:
Defaults can be setup in Drupal, too, if you want to save hard coding it (admins would see this default text initially and change if the wanted to).